08-kogito-travel-agency/src/main/java/org/kie/kogito/app/ProcessEventListenerConfig.java

```diff
@@ -0,0 +1,14 @@
+package org.kie.kogito.app;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.kie.addons.monitoring.process.PrometheusProcessEventListener;
```

Comment:
if you change to 0.8.1
```suggestion
import org.kie.kogito.monitoring.process.PrometheusProcessEventListener;
```